assets/less/cds-rdm/globals/site.overrides

```diff
@@ -53,4 +53,46 @@
 
 .info-box {
   padding: @defaultPadding !important;
+}
+
+#records-list {
+
+  width: 100%;
+
+  h3, h4{
+    font-weight: normal;
+  }
+
+  .ui.grid {
+    margin: 0;
+
+    .column {
+      padding: 0;
```

Comment:
Minor: Same as above but with `p-0`.

Reply:
I don't think we are saving lines if we have to a add the p-0 in multiple lines in the html.

Reply:
we are saving the amount of CSS we need to maintain and decrease the complexity of the styling sheet. It is not always about the number of the lines.